Microsoft.Toolkit.Win32/Microsoft.Toolkit.Forms.UI.XamlHost/Microsoft.Toolkit.Forms.UI.XamlHost.csproj

```diff
@@ -0,0 +1,80 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net462</TargetFramework>
+    <RootNamespace>Microsoft.Toolkit.Forms.UI.XamlHost</RootNamespace>
+    <AssemblyName>Microsoft.Toolkit.Forms.UI.XamlHost</AssemblyName>
+    <GenerateAssemblyInfo>true</GenerateAssemblyInfo>
+    <RunCodeAnalysis>true</RunCodeAnalysis>
+
+    <Title>Windows Community Toolkit XAMLHost for Windows Forms</Title>
+    <Description>This library provides XAML islands interop helpers for Windows Forms. It is part of the Windows Community Toolkit.</Description>
+    <PackageTags>XAML Islands Win32 Forms WindowsForms WinForms XamlHost</PackageTags>
+
+    <!--
+    Need class WIndows.UI.Xaml.Hosting.DesktopWindowXamlSource, which was introduced in Insiders SDK 17692 on 19-Jun 2018
+    -->
+    <TargetPlatformMinVersion>10.0.17692.0</TargetPlatformMinVersion>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
```

Comment:
This property group looks like it should be nuked; probably added by editing the properties in the VS UI?